app/assets/stylesheets/application/donations/donations.scss

```diff
@@ -63,6 +88,17 @@
   }
 }
 
+#new_donation {
+  .radio label {
+    width: 205px;
+    .tip-amount {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4